README.md

```diff
@@ -198,7 +198,20 @@ IPU ES2K target.
    git checkout ipdk_v23.07
    ```
 
-3. Build P4-K8s binaries and container images.
+3. Update `inframanager/config.yaml` file.
+
+   Refer to section [inframanager/config.yaml file update](#inframanagerconfigyaml-file-update)
+   for details.
+   All fields have a default value in the file. Please verify if the values
+   correspond to the desired values especially arp-mac.
+   This should be the MAC of the interface which user intends to select in the
+   step 2
+   of the [Deploy P4 Kubernetes section](#deploy-p4-kubernetes) for ARP proxy gateway.
+   Any other changes to the config.yaml file if needed needs to go before the
```

Comment:
This statement is present in 3 places. Here, 300 and 349. @nupurjai @dhruvpt  Please check and recommend if we should remove any of the occurrences ?